springwolf-core/src/test/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/annotation/AsyncPublisherAnnotationScannerTest.java

```diff
@@ -66,7 +66,7 @@ public void scan_componentHasPublisherMethod() {
         Message message = Message.builder()
                 .name(SimpleFoo.class.getName())
                 .title(SimpleFoo.class.getSimpleName())
-                .description(null)
+                .description("")
```

Comment:
This is one of the issues I found because Message only implemented equals with the name